app/assets/stylesheets/apply.scss.erb

```diff
@@ -1,45 +1,86 @@
+.card-style {
+  background-color: white;
+  box-shadow: 0 0 5px 2px #ddd;
+  border: 1px solid #bbb;
+}
+
 .panel {
-  height: 100vh;
-  text-align:center;
+  top: 0;
   width:100%;
-  min-height:500px;
+  height:500px;
+  position: fixed;
+  z-index: -1;
+
+  .panel-container-left &, .panel-container-right & {
+    width: 50%;
+    height: 100vh;
+    @media only screen and (max-width: 45em) {
+      position: absolute;
+      top: initial;
+      width: 100%;
+    }
+  }
+
+  .panel-container-left & {
+    float: left;
+  }
+
+  .panel-container-right & {
```

Comment:
That actually does something different: http://thesassway.com/intermediate/referencing-parent-selectors-using-ampersand